upbound_uxp/datadog_checks/upbound_uxp/check.py

```diff
@@ -0,0 +1,590 @@
+#!python3
+
+# Upbound Universal Crossplane DataDog Integration
+
+# This integration monitors Upbound Universal Crossplane
+# and provider Kubernetes pods.
+
+import json
+import os
+import re
+import subprocess
+import sys
+
+import requests
+from kubernetes import client, config
+from prometheus_client.parser import text_string_to_metric_families
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+
+# Agent Behavior
+#
+# When no metrics are specified, a predefined min set is used and
+# there are no pod annotations required for the Crossplane and Provider
+# pods.
+#
+# None, Min and More metrics sets can receive additions from the
+# metrics section in the DataDog configurations yaml file.
+#
+# When pod annotations are not ignored, they will be preferred and
+# overwrite any other metrics selections.
+
+__version__ = "0.0.1"
```

Comment:
This is probably not needed, `__version__` is included in `__about__.py` already.

Reply:
Removed version from check.py.